napari/viewer.py

```diff
@@ -38,3 +47,45 @@ def __init__(self, *images, meta=None, multichannel=None, clim_range=None,
         for name, image in named_images.items():
             self.add_image(image, meta=meta, multichannel=multichannel,
                            clim_range=clim_range, name=name)
+        self.theme = 'dark'
+
+    @property
+    def theme(self):
+        """string: Color theme.
+        """
+        try:
+            return self._theme
+        except AttributeError:
+            return None
+
+    @theme.setter
+    def theme(self, theme):
+        if self.theme is not None and theme == self.theme:
+            return
+        self._theme = theme
+
+        if theme not in palettes.keys():
+            raise KeyError("Theme '%s' not found, options are %s."
+                           % (theme, list(palettes.keys())))
+
+        palette = palettes[theme]
+
+        # template and apply the primary stylesheet
+        themed_stylesheet = template(self.raw_stylesheet, **palette)
+        self._qtviewer.setStyleSheet(themed_stylesheet)
+
+        # set window styles which don't use the primary stylesheet
+        self.window._status_bar.setStyleSheet('QStatusBar { background: %s;'
+            'color: %s}' % (palette['background'], palette['text']))
```

Comment:
fix this pep8 error and we'll be ready to merge :)

Reply:
fixed! merge away!